Makefile

```diff
@@ -1,16 +1,22 @@
-.PHONY: all image package dist clean
+.PHONY: all image package dist deploy clean
 
-all: package
+all: image dist
 
 image:
 	docker build --tag amazonlinux:nodejs .
 
-package: image
-	docker run --rm --volume ${PWD}/lambda:/build amazonlinux:nodejs npm install --production
+awsconfigure:
+	docker run -it --rm --volume ${PWD}/../.aws:/root/.aws amazonlinux:nodejs aws configure
```

Comment:
@jorrit while writing a comment about this debatable path I realized: you cannot mount `~/.aws` because it expands to something like `/home/username` and this path normaly isn't available as VirtualBox shared folder inside the virtual machine. You only have a path like `/c/Users/username` available and not `/home/username`